lib/data_hygiene/publishing_api_sync_check.rb

```diff
@@ -57,6 +59,8 @@ def initialize(scope)
           Whitehall.url_maker.polymorphic_path(model)
         end
       end
+
+      add_translations_expectation
```

Comment:
OK, I've added a condition to this statement so that the expectation is only added if the first item in the scope can have a translation.